sunpy/net/vso/vso.py

```diff
@@ -792,3 +792,70 @@ def _attrs_module(cls):
 
     def __del__(self):
         self.api.transport.session.close()
+
+    @classmethod
+    def register_values(cls):
+        # We always use the local file for now.
+        return cls.get_vso_values(load=True)
+
+    @staticmethod
+    def get_vso_values(load=False, save=False):
```

Comment:
same here about save and load being separate.

Reply:
Fixed.